lib/Backends/CPU/libjit/libjit_conv.cpp

```diff
@@ -113,21 +113,23 @@ void libjit_convDKKC8_foreach_xy_filter_pixels(
     unsigned depthStrips, unsigned sizeGroupY, size_t numChannels, float *outW,
     const float *inW, const float *filterW, const float *biasW,
     const size_t *outWdims, const size_t *inWdims, const size_t *filterWdims,
-    const size_t *biasWdims, size_t filterSize, size_t stride, size_t pad,
+    const size_t *biasWdims, size_t filterSize, size_t stride, size_t *pads,
     size_t group) {
   // The loops below look scary but the the idea is simple. We iterate over
   // the pixels in the output tensor and calculate the coordinate of the source
   // tensor. When we process the Y row we try to process [sizeGroupY] elements
   // at once. After we finish the row we handle the odd cases by handling one y
   // value at a time.
 
+  size_t pad_t = pads[0];
+  size_t pad_l = pads[1];
```

Comment:
Just for the sake of readability, could we use a temporary shapeTLBR object here, instead of hardcoding top == pads[0] and pad_l == pads[1]?

Reply:
@qcolombet Actually, I planed to do that. However, the include/glow/Base/Type.h is not included in  libjit_conv.cpp. I noticed that ShapeNHWC etc. are not used here and I assume here we prefer not to use Glow defined data type. Please let me know if it is not the case :)

Reply:
Good point.
Changing this kind of dependencies is definitely out of the scope of this PR.
Let us go back to this in a follow up PR.